### PR TITLE
Explorer

### PR DESCRIPTION
**Pull request by johnalotoski**
* Adds explorer bare metal machine via equinix for a high IOPS backend pool member

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -80,16 +80,38 @@
       flake = false;
     };
 
-    #cardano-rosetta = {
-    #  url = "github:input-output-hk/cardano-rosetta";
-    #  flake = false;
-    #};
+    # cardano-rosetta = {
```
Comment (disassembler) [inline flake.nix:83]: why are we leaving this in?

File: nix/cardano/alerts.nix [hunk 2]
```diff
@@ -253,4 +253,90 @@ in
...
+      {
+        alert = "varnish cache too small or ttl too long";
+        expr = ''rate(varnish_main_n_lru_nuked[30m]) > 5 * rate(varnish_main_n_expired[30m])'';
+        for = "1h";
+        labels.severity = "page";
+        annotations = {
+          summary = "{{$labels.alias}}: Too many objects (5 times the number of expiring objects) are being forcefully evicted from varnish cache due to memory constraints.";
+          description = "{{$labels.alias}}: Consider increasing varnish malloc limit or decreasing beresp.ttl";
+        };
+      }
```
Comment (disassembler) [inline nix/cardano/alerts.nix:339]: we probably should have something for db-sync height compared to node tip

**Review by disassembler — CHANGES_REQUESTED** [review]

File: nix/cardano/alerts.nix (hunk 2 above)
Comment (johnalotoski) [reply nix/cardano/alerts.nix:339]: Yup we're covered there with the db-sync alert attr set above.

File: flake.nix (hunk 1 above)
Comment (johnalotoski) [reply flake.nix:83]: It was a placeholder that can go -- removing.

**Review by disassembler — APPROVED** [review]

**Pull request merged**